desktop/cmp/input/DateInput.js

```diff
@@ -215,30 +215,32 @@ export class DateInput extends HoistInput {
             inputHasFocus = this.containsElement(activeEl);
 
         if (!popoverHasFocus && !inputHasFocus) {
+            this.setPopoverOpen(false);
```

Comment:
This change appears to be breaking the actual flushing of a just-clicked value from the popover into the control.  Need to investigate further....